appengine/rails-cloudsql-postgres/Gemfile

```diff
@@ -61,9 +61,9 @@ end
 gem "tzinfo-data", platforms: [:mingw, :mswin, :x64_mingw, :jruby]
 
 # [START gems]
-# Added at 2017-08-23 15:24:52 -0700 by franknatividad:
-gem "pg", "~> 0.21.0"
+# Added at 2021-04-20 by trambui09:
+gem "pg", "~> 1.2"
 
-# Added at 2017-08-07 11:54:12 -0700 by USER:
-gem "appengine", "~> 0.4.1"
+# Added at 2021-04-20 by trambui09:
+gem "appengine", "~> 0.6.0"
```

Comment:
Use `~> 0.6` instead of `~> 0.6.0`.

The former lets the gem update to later minor versions (like 0.7, 0.8). The latter pins it to 0.6.x and doesn't allow 0.7 etc.